setup.py

```diff
@@ -74,43 +76,59 @@ def md2pypi(filename):
     return content
 
 
+def pip(filename):
+    """Parse pip reqs file and transform it to setuptools requirements."""
+    requirements = []
+    for line in open(os.path.join('requirements', filename)):
+        line = line.strip()
+        if not line or '://' in line or line.startswith('#'):
+            continue
+        match = RE_REQUIREMENT.match(line)
+        if match:
+            requirements.extend(pip(match.group('filename')))
+        else:
+            requirements.append(line)
+    return requirements
+
+
 long_description = '\n'.join((
     md2pypi('README.md'),
     md2pypi('CHANGELOG.md'),
     ''
 ))
 
+install_requires = pip('install.pip')
+tests_require = pip('test.pip')
 
-install_requires = []
-tests_require = []
 
 setup(
     name='udata-piwik',
-    version='0.9.4.dev',
+    version='0.9.4-dev',
     description='Piwik support for uData',
     long_description=long_description,
     url='https://github.com/opendatateam/udata-piwik',
-    author='Opendata Team',
-    author_email='opendatateam@data.gouv.fr',
-    packages=find_packages(),
+    author='OpenDataTeam',
+    author_email='contact@opendata.team',
+    packages=['udata_piwik'],
     include_package_data=True,
     install_requires=install_requires,
     tests_require=tests_require,
+    extras_require={
+        'test': tests_require,
+    },
     license='LGPL',
-    use_2to3=True,
-    keywords='udata piwik',
+    zip_safe=False,
+    keywords='udata, piwik',
```

Comment:
setuptools and pypi require a space as separator but for this to work properly, you need to remove the `from __future__ import unicode_literals` line